### ISSUE TITLE
should a frame navigated to about:blank inherit a service worker controller?

### ISSUE THREAD

**Issue by wanderview** [w3c/ServiceWorker#1350]
Currently we inherit a service worker controller from the parent frame when a srcdoc or initial about:blank frame is created.  Should some form of inheritance happen when an iframe with a non-blank URL gets navigated to about:blank?

If so, where should the service worker be inherited from.  Apparently some browsers inherit the origin from the navigation initiator instead of the parent.  Should service worker inheritance match?  (This is a similar question to #1261).

This question was raised in https://crbug.com/880768.

**Comment by annevk** [comment]
Yes, and where I guess depends on how we generally resolve those questions for such a document, as this is not the only thing it'll inherit. Unfortunately all of that is still a mess in the HTML Standard.

**Comment by monica-ch** [comment]
[Bug triage]: Needs dicussion
